oops/oops_test.go

```diff
@@ -686,35 +686,34 @@ func BenchmarkErrorf(b *testing.B) {
 	}
 }
 
-
 func BenchmarkWrapf(b *testing.B) {
-	benchmarkCases := []struct{
+	benchmarkCases := []struct {
```

Comment:
Not sure why this formatting differs between our editors 🤔

Reply:
Goland did this for me 🤷